multisig/simulations/ethereum/vip-026/index.ts

```diff
@@ -0,0 +1,296 @@
+import { impersonateAccount } from "@nomicfoundation/hardhat-network-helpers";
+import { expect } from "chai";
+import { BigNumber, Contract } from "ethers";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { checkIsolatedPoolsComptrollers } from "../../../../src/vip-framework/checks/checkIsolatedPoolsComptrollers";
+import { checkVToken } from "../../../../src/vip-framework/checks/checkVToken";
+import { checkInterestRate } from "../../../../src/vip-framework/checks/interestRateModel";
+import {
+  checkRewardsDistributor,
+  checkRewardsDistributorPool,
+} from "../../../../src/vip-framework/checks/rewardsDistributor";
+import { forking, pretendExecutingVip } from "../../../../src/vip-framework/index";
+import {
+  COMPTROLLER,
+  FRAX,
+  REWARDS_DISTRIBUTOR_XVS,
+  VTOKEN_RECIEVER,
+  XVS,
+  XVS_REWARD_TRANSFER,
+  sFRAX,
+  vFRAX,
+  vip026,
+  vsFRAX,
+} from "../../../proposals/ethereum/vip-026";
+import POOL_REGISTRY_ABI from "./abi/PoolRegistry.json";
+import RESILIENT_ORACLE_ABI from "./abi/ResilientOracle.json";
+import REWARD_DISTRIBUTOR_ABI from "./abi/RewardsDistributor.json";
+import COMPTROLLER_ABI from "./abi/comptroller.json";
+import ERC20_ABI from "./abi/erc20.json";
+import VTOKEN_ABI from "./abi/vToken.json";
+
+const { ethereum } = NETWORK_ADDRESSES;
+const BLOCKS_PER_YEAR = BigNumber.from(2628000); // assuming a block is mined every 12 seconds
+
+interface RiskParameters {
+  borrowCap: string;
+  supplyCap: string;
+  collateralFactor: string;
+  liquidationThreshold: string;
+  reserveFactor: string;
+  initialSupply: string;
+  vTokenReceiver: string;
+}
+
+const vFrax_riskParameters: RiskParameters = {
+  borrowCap: "8000000",
+  supplyCap: "10000000",
+  collateralFactor: "0.75",
+  liquidationThreshold: "0.80",
+  reserveFactor: "0.1",
+  initialSupply: "5000",
+  vTokenReceiver: ethereum.VTREASURY,
```

Comment:
```suggestion
  vTokenReceiver: "0x6e74053a3798e0fC9a9775F7995316b27f21c4D2",
```

Reply:
I think we don't use this field... Maybe we could remove it